Box.V2/Models/BoxZip.cs

```diff
@@ -0,0 +1,42 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Box.V2.Models
+{
+    /// <summary>
+    /// Box representation of a created zip
+    /// </summary>
+    public class BoxZip
```

Comment:
Can you please refactor the name to BoxZipInfo to match the [Box Java SDK](https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxZipInfo.java#L16)?